server/controllers/user.controller.js

```diff
@@ -115,11 +138,146 @@ const UserController = function () {
     }
   };
 
+  var sendEmail = async function (req, res, next) {
+    try {
+      const apiKey = process.env.BREVO_API_KEY;
+      const url = "https://api.brevo.com/v3/smtp/email";
+
+      const emailType = req.body.emailType;
+      const typeformLink = req.body.formLink || "";
```

Comment:
nit: since we already removed all typeform dependencies, maybe rename this to `formLink`? 
```suggestion
      const formLink = req.body.formLink || "";
```

Reply:
agree

Reply:
Changed.